File: emulinker/src/main/java/org/emulinker/kaillera/model/event/ConnectedEvent.java
package org.emulinker.kaillera.model.event;

import org.emulinker.kaillera.model.*;

public class ConnectedEvent implements UserEvent {
  private KailleraUser user;
  private KailleraServer server;

  public ConnectedEvent(KailleraServer server, KailleraUser user) {
    this.server = server;
    this.user = user;
  }

  public String toString() {
    return "ConnectedEvent";
  }

  public KailleraUser getUser() {
    return user;
  }

  public KailleraServer getServer() {
    return server;
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/model/event/ConnectedEvent.kt
package org.emulinker.kaillera.model.event

import org.emulinker.kaillera.model.KailleraServer
import org.emulinker.kaillera.model.KailleraUser

data class ConnectedEvent(val server: KailleraServer, override val user: KailleraUser) : UserEvent {
  override fun toString(): String {
    return "ConnectedEvent"
  }
}
